composition_stats/tests/test_composition.py

```diff
@@ -231,6 +231,14 @@ def test_clr(self):
         clr(closure(self.cdata2))
         npt.assert_allclose(self.cdata2, np.array([2, 2, 6]))
 
+        # test ignore_zero
+        with self.assertWarnsRegex(RuntimeWarning, 'divide by zero encountered'):
+            cmat = clr(closure(self.cdata3), ignore_zero=False)
+            assert np.any(np.isfinite(cmat))
```

Comment:
Should this not `assert not np.all(np.isfinite(cmat))`?

Reply:
True! I've modified accordingly